cmd/storage-datatypes.go

```diff
@@ -37,6 +37,13 @@ type RenameOptions struct {
 	BaseOptions
 }
 
+// DiskInfoOptions options for requesting custom results.
+type DiskInfoOptions struct {
+	DiskID  string `msg:"id"`
+	Metrics bool   `msg:"m"`
+	NoOp    bool   `msg:"np"`
```

Comment:
What does NoOp mean in this context?

Reply:
Do not do any operation on the disk and only capture in-memory information